src/AstroBase.jl

```diff
@@ -115,4 +115,78 @@ function tio_locator(jd1, jd2)
     -47e-6 * t * sec2rad(1)
 end
 
+"""
+    normalize_pi_angle(a)
+
+Returns an angle(radians) in the range [-π, π) for a given angle(radians).
+
+# Example
+
+```jldoctest
+julia> normalize_pi_angle(21)
+2.1504440784612413
+```
+"""
+function normalize_pi_angle(a)
+    w = mod(a, 2pi)
+    if abs(w) >= pi w -= (a < 0.0? -abs(w): abs(w)) end
```

Comment:
Apart from @giordano's comment above. Oneliners like this are a no-go. There are several branches and a mutation of a variable on one line. This is like asking for trouble...